src/Layout/tool_fsharp/tool_fsc.csproj

```diff
@@ -5,13 +5,26 @@
     <ServerGarbageCollection>true</ServerGarbageCollection>
   </PropertyGroup>
 
+  <!-- MIBC, PGO -->
+  <Import Project="..\redist\targets\GetRuntimeInformation.targets" /> <!-- Rid is set here -->
+  <PropertyGroup>
+    <MibcPath>$(PkgMicrosoft_FSharp_Compiler)/contentFiles/mibc</MibcPath>
+    <MibcFile>$(MibcPath)/optimization.$(Rid).mibc.runtime/DotNet_FSharp.mibc</MibcFile>
```

Comment:
For Mac, it will try to find file for macOS rid, and won't find it (we use Linux one for Mac). So we either have to pack Linux one with macOS name or use Linux one for macOS here too.

Reply:
Yeah I will adjust this once this proves to be a valid approach at all.

Reply:
Actually it looks like Rid is set to linux-x64 on MacOS, I checked the binlog. So that's probably ok. Otherwise, this would explode anyway, since the files wouldn't be found.